third_party/move/move-vm/test-utils/src/storage.rs

```diff
@@ -158,7 +163,7 @@ impl<'a, 'b, S: MoveResolver> DeltaStorage<'a, 'b, S> {
 /// Simple in-memory storage for modules and resources under an account.
 #[derive(Debug, Clone)]
 struct InMemoryAccountStorage {
-    resources: BTreeMap<StructTag, Bytes>,
+    resources: BTreeMap<StructTag, BytesWithLayout>,
```

Comment:
I think this is an irrelevant change? Do we need this?

Reply:
In bad_storage_tests.rs line 108 and a few other tests, we apply ChangeSet outputted by session to InMemoryStorage. So, InMemoryStorage had to take Changes<Bytes, (Bytes, MoveTypeLayout)> as input.